server.py

```diff
@@ -432,6 +432,32 @@ def put(self, region, id):
             tournament.regions = args['regions']
 
         dao.update_tournament(tournament)
+        return {"success": True}
```

Comment:
Actually I put PUT should return the entire updated object. I guess I overlooked this to begin with. You can leave it like this for now but can you throw in a `# TODO return tournament`?